cmd/osbuild-worker/main.go

```diff
@@ -128,6 +128,9 @@ func setProtection(protected bool) {
 		}
 		return
 	}
+	if len(asInstanceOutput.AutoScalingInstances) == 0 {
+		logrus.Info("No Autoscaling instace is defined ")
```

Comment:
```suggestion
		logrus.Info("No Autoscaling instace is defined")
```

More importantly, there's no `return`.

Reply:
D'oh! Thanks for that 🤦🏼‍♂️. Fixed